docs/getting-started/concepts.md

```diff
@@ -3,19 +3,40 @@ template: main.html
 ---
 
 # Iter8
-Iter8 is a metrics-driven release optimizer built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well, are secure, and maximize business value.
+Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that Kubernetes apps and ML models perform well and maximize business value.
 
 ## Iter8 experiment
 <p align='center'>
   <img alt-text="load-test-http" src="../../images/iter8-intro-dark.png" width="70%" />
 </p>
 
-Iter8 experiments make it simple to collect performance, risk, and business metrics for apps and ML models, assess, compare and validate one or more app/ML model versions, promote the winning version, and maximize business value in each release.
+Iter8 experiments make it simple to collect performance and business metrics for apps and ML models, assess and compare one or more app/ML model versions, validate [service-level objectives (SLOs)](#service-level-objectives), promote the winning version, and maximize business value during each release.
 
-## Experiment chart
-Iter8 experiment charts are [Helm charts](https://helm.sh/docs/topics/charts/) under the covers, and enable simple, declarative, and reusable metrics-driven experiments. Iter8 combines experiment charts with user supplied values to generate runnable `experiment.yaml` files or Kubernetes experiment manifests. The former is used for running experiments in the local environment, while the latter is used for running experiments inside Kubernetes.
+### Tasks
+An experiment is a set of tasks that are executed in a specific sequence. Iter8 provides pre-defined tasks for various functions such as the following:
+
+1. Generating load and collecting Iter8's built-in metrics for HTTP services.
+2. Generating load and collecting Iter8's built-in metrics for gRPC services.
+3. Fetching metrics for one or more versions of an app from databases. 
```

Comment:
I wonder if we can/should be more explicit: from metrics databases.  _or_ from REST accessible metrics stores.

Reply:
Fixed.